### PR TITLE
add method to check tcp for capaiblities

### PR DESCRIPTION
**Pull request by rivernate**

### CODE REVIEW

File: lib/ims/lti/services/tool_consumer_profile_service.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module IMS::LTI::Services
+  class ToolConsumerProfileService
+
+    attr_accessor :tcp
+
+    def initialize(tool_consumer_profile)
+      @tcp = tool_consumer_profile
+    end
+
+    def supports_capabilities?(capability, *capabilities)
```
Comment (westonkd) [inline lib/ims/lti/services/tool_consumer_profile_service.rb]: Is there a downside of changing the signature to 
```
supports_capabilities?( *capabilities)
```
and dropping `capabilities.unshift(capability)` from line 11?

Comment (rivernate) [inline lib/ims/lti/services/tool_consumer_profile_service.rb]: If you just use a single splat argument the method takes 0-* arguments.  The way it's no the method take 1-* arguments.

**Comment by rivernate** [comment]
Test comment

**Comment by rivernate** [comment]
@channel test comment

**Comment by rivernate** [comment]
:magic-linux: test comment

**Pull request merged**